src/cs/usfca/edu/histfavcheckout/controller/HistFavCheckoutHandler.java

```diff
@@ -252,7 +253,37 @@ public OperationalResponse checkout(OperationalRequest request) {
 		resp.setConfirm(true);
 		return resp;
 	}
-	
+
+	public ResponseEntity<?> returnMovie(int userId, int movieId) {
+		Optional<User> user = userRepository.findById(new PrimaryKey(userId, movieId));
+		Optional<Inventory> inventory = inventoryRepository.findById(movieId);
+		OperationalResponse confirm = new OperationalResponse();
+		if(user.isPresent() && inventory.isPresent()) {
+			User u = user.get();
+			if (u.isCheckouts() && userRepository.updateCheckoutReturn(u.getId(), getCurrentDate()) == 1) {
+				Inventory record = inventory.get();
+				record.setAvailableCopies(record.getAvailableCopies() + 1);
+				if (inventoryRepository.updateAvailableCopies(record.getAvailableCopies(), record.getProductId()) == 1) {
+					confirm.setConfirm(true);
+					confirm.setMessage("Movie returned successfully");
+					System.out.println("SUCCESS");
+					return ResponseEntity.status(HttpStatus.OK).body(confirm);
```

Comment:
Remove print statements. You can put a todo or comment mentioning that this needs to be logged